HW_1/Sem_1.py

```diff
@@ -0,0 +1,92 @@
+#         print(f'{y} x {x:2} = {x * y:2}\t', end='     ')
+#     print('  ')
+# ------------------------------------------------------------------------------------------------------------------------------------
+# 2) Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+# Дано a, b, c - стороны предполагаемого треугольника.
+# Требуется сравнить длину каждого отрезка-стороны с суммой двух других.
+# Если хотя бы в одном случае отрезок окажется больше суммы двух других,
+# то треугольника с такими сторонами не существует. Отдельно сообщить является ли треугольник разносторонним,
+# равнобедренным или равносторонним.
+
+# a = int(input("Введите сторону А: "))
+# b = int(input("Введите сторону B: "))
+# c = int(input("Введите сторону C: "))
+# EQUILATERAL = "равносторонний"
+# ISOSCELES = "равнобедренный"
+# VERSATILE  = "разносторонний"
+
+# if a < b + c and b < a + c and c < a + b:
+#     if a == b and a == c:
+#         print("Треугольник существует и он " + EQUILATERAL)
+#     elif a == b or b == c:
+#         print("Треугольник существует и он " + ISOSCELES)
+#     else:
+#         print("Треугольник существует и он " + VERSATILE )
+# else:
+#     print("Треугольник не существует")
+
+# ----------------------------------------------------------------------------------------------------------------------------------------
+# 3) Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+# Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+
+# number = int(input("Введите число от 1 до 100 000: "))
+# LOWER_LIMIT = 1
+# UPPER_LIMIT = 100000
+
+# if LOWER_LIMIT < number < UPPER_LIMIT:
+#     for i in range(2, int(number ** 0.5) + 1):
+#         if (number % i) == 0:
+#             print(number, "число составное")
+#             break
+#     else:
+#         print(number, "число простое")
+# else:
+#     print(number, "вы ввели отрицательное число или число больше 100 000")
+
+# --------------------------------------------------------------------------------------------------------------------------------------------
+# 4) Программа загадывает число от 0 до 1000. Необходимо угадать число за 10 попыток.
+# Программа должна подсказывать «больше» или «меньше» после каждой попытки.
+# Для генерации случайного числа используйте код:
+# from random import randint
+# num = randint(LOWER_LIMIT, UPPER_LIMIT)
+
+from random import randint
+
+LOWER_LIMIT = 0
+UPPER_LIMIT = 1000
+count = 9
+num = randint(LOWER_LIMIT, UPPER_LIMIT)
+
+while count != -1:
+    number = int(input("Введите число от 0 до 1000: "))
+    if number == num:
+        print("Вы угадали число - это ", number)
+        break
+    if number < num:
+        print ("Загаданное число больше ", number)
+    else:
+        print("Загаданное число меньше ", number)
+    print(' ')
```

Comment:
Можно просто пустой принт
```suggestion
    print()
```